--- tools/build_defs/shell_toolchain/toolchains/toolchain_mappings.bzl
+++ tools/build_defs/shell_toolchain/toolchains/toolchain_mappings.bzl
@@ -11,21 +11,18 @@ TOOLCHAIN_MAPPINGS = [
     ToolchainMapping(
         exec_compatible_with = [
             "@bazel_tools//platforms:linux",
-            "@bazel_tools//platforms:x86_64",
         ],
         file = "@rules_foreign_cc//tools/build_defs/shell_toolchain/toolchains/impl:linux_commands.bzl",
     ),
     ToolchainMapping(
         exec_compatible_with = [
             "@bazel_tools//platforms:windows",
-            "@bazel_tools//platforms:x86_64",
         ],
         file = "@rules_foreign_cc//tools/build_defs/shell_toolchain/toolchains/impl:windows_commands.bzl",
     ),
     ToolchainMapping(
         exec_compatible_with = [
             "@bazel_tools//platforms:osx",
-            "@bazel_tools//platforms:x86_64",
         ],
         file = "@rules_foreign_cc//tools/build_defs/shell_toolchain/toolchains/impl:osx_commands.bzl",
     ),
